lib/config_parser.c

```diff
@@ -329,7 +329,7 @@ static int cp_add_global_guest_account(gpointer _v)
 
 	user = usm_lookup_user(_v);
 	if (!user) {
-		pr_err("Fatal error: unable to find `%s' account.\n",
+		pr_err("Fatal error: unable to find `%p' account.\n",
 			_v);
```

Comment:
No. user account is a string. %s is how you print a string.

-ss

Reply:
It's a void pointer. Should be cast to char* I guess.